task1/task1.js

```diff
@@ -0,0 +1,86 @@
+// 1. Write a JavaScript function that takes two numbers as parameters and returns their sum.
+let func1 = (a, b) => a + b;
+
+console.log("func1(2, 3) = " + func1(2, 3));
+
+// 2. Write a JavaScript function that checks if a variable is of type 'string'
+// and returns true if it is, false otherwise.
+
+let func2 = (a) => typeof a === "string";
+
+console.log('func2("Some String") = ' + func2("Some String"));
+console.log("Primitive number func2(5) = " + func2(5));
+console.log("Primitive boolean func2(true) = " + func2(true));
+console.log("arrow function func2(func1) = " + func2(func1));
+console.log("Object func2({}) = " + func2({}));
+console.log("func2(undefined) = " + func2(undefined));
+
+// 3. Write a JavaScript function that checks if a given value is a primitive data type
+// (number, string, boolean, null, or undefined) and returns true if it is, false otherwise.
+
+let func3 = (a) =>
+  a === null ||
+  a === undefined ||
+  typeof a === "number" ||
+  typeof a === "boolean" ||
+  typeof a === "string";
```

Comment:
FYI, the task missed 'symbol' and 'bigint' as the primitives.
https://developer.mozilla.org/en-US/docs/Glossary/Primitive

Reply:
There is might be a check for all like
```
a === Object(a)
```